setup.py

```diff
@@ -21,14 +21,15 @@
         'pandas',
         'scipy',
         'tifffile',
-        'torch==0.4',
+        'torch==1.0',
         'tqdm',
     ],
     extras_require={
         'dev': [
             'flake8',
             'pylint',
             'pytest',
+            'pytest-cov',
```

Comment:
You may also want `pytest-raises`. It allows for nice functionality of having tests that you know will raise an error.

Example:
https://github.com/AllenCellModeling/aics-deformation/blob/master/aicsdeformation/tests/test_aicsdeformation.py#L35